http-server/src/main/java/io/airlift/http/server/HashLoginServiceProvider.java

```diff
@@ -42,18 +37,9 @@ public HashLoginServiceProvider(HttpServerConfig config)
     public HashLoginService get()
     {
         String authConfig = config.getUserAuthFile();
-        try {
-            if (!isNullOrEmpty(authConfig)) {
-                HashLoginService service = new HashLoginService(HttpServerModule.REALM_NAME, authConfig);
-                service.loadUsers();
-                return service;
-            }
-            return null;
+        if (!isNullOrEmpty(authConfig)) {
+            return new HashLoginService(HttpServerModule.REALM_NAME, authConfig);
         }
-        catch (IOException e) {
```

Comment:
`loadUsers()` was throwing previously and it's removed in this release, so nothing is throwing an IOE now. BTW it was already a noop in `9.3.13.M0`.